lib/cloud/aws/policy_statements.go

```diff
@@ -56,7 +56,7 @@ func StatementForECSManageService() *Statement {
 		Effect: EffectAllow,
 		Actions: []string{
 			"ecs:DescribeClusters", "ecs:CreateCluster", "ecs:PutClusterCapacityProviders",
-			"ecs:DescribeServices", "ecs:CreateService", "ecs:UpdateService",
+			"ecs:DescribeServices", "ecs:CreateService", "ecs:UpdateService", "ecs:ListServices",
```

Comment:
We are going to need this to implement the Auto-Upgrade for those services.
Currently, the Auto-Upgrade service only expects a single ECS Service on the cluster.
However, this PR adds multiple ECS Services, so it won't work.
A follow up PR will address that.

To get multiple ECS Services from a ECS Cluster without their name, we must use the ListServices.
(DescribeServices requires the Services name as input).

Given that the Auto-Upgrader runs unattended, it has no way to ask the user to add permissions in case it lacks any.
Adding this new permission now ensures we have the required permissions and are ready to implement the Auto-Upgrader for multiple ECS Services.